kytos/core/napps/manager.py

```diff
@@ -95,6 +94,11 @@ def install(self, napp_uri, enable=True):
 
         log.info("New NApp installed: %s", napp)
 
+        napp = NApp.create_from_json(dst/'kytos.json')
+        log.info('Installing Napp dependencies:')
+        for napp in napp.napp_dependencies:
+            self.install(napp, enable)
```

Comment:
The `install()` method receives, as argument, a `napp_uri` and, optionally, a `enable` (boolean) arg.

Here you are passing `napp` (NApp object) instead of the `napp_uri`. Is this right?